### DISCUSSION TITLE
Bucket version / replication on Minio Gateway NAS ?

### DISCUSSION THREAD

**Discussion by jo-sanchez** [minio/minio#11709] [Q&A] [answered]
Hi,
Is there a clean way to use Bucket versioning and replication between two Minio gateways NAS (with NAS backhand storage, then).
I understand that bucket versioning depends on erasure coding. Which doesn't seem to be handled by minio when using  gateway nas.

I could still use erasure coding in server mode with docker volumes pointing to NFS shares instead of running minio in gateway nas mode but does it make sense to implement it like that since I don't need storage redundancy because it is already handled by the NAS bay.

Thanks.

**Comment by harshavardhana** [comment, marked as answer]
Bucket versioning and replication will not be supported on gateways and don't use NFS with erasure-coded backend - it is unsupported and will not work properly.

Erasure coding requires that provide us regular drives.
